### PR TITLE
update yargs to v21.1.0

### PR DESCRIPTION
**Pull request by Uzlopak**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Uzlopak** [comment]
@Eomm

File: lib/args.js [hunk 1]
```diff
@@ -81,28 +82,32 @@ module.exports = function parsedArgs (args) {
...
     tag: parsedArgs.tag,
     verbose: parsedArgs.verbose,
     semver: parsedArgs.semver,
-    major: parsedArgs.major
+    major: toBool(parsedArgs.major)
   })
 }
 
+function toBool (value) {
+  return value && Boolean(value)
```
Comment (Eomm) [inline lib/args.js:110]: How the tests are passing? 🤔 

```
Boolean('false') // true
```

Comment (Uzlopak) [reply lib/args.js:110]: Black magic. I wonder if yargs is always string true or false or not. Will have to investigate it furthee

Comment (Uzlopak) [reply lib/args.js]: Ok, fixed it. the uit test was wrong.

**Review by Eomm — APPROVED** [review]

**Pull request merged**